server/server.go

```diff
@@ -2631,8 +2631,10 @@ func (s *Server) setFirstPingTimer(c *client) {
 			d = firstClientPingInterval
 		}
 	}
+
```

Comment:
nitpick: could remove this line and the other below so there is no actual changes in that function..